src/App/screens/Instructor/components/Nav/components/NavLink.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+
+const NavLink = ({
+  route,
+  isActive,
+  onClick,
+  href,
+}) => (
+  <a
+    href={href}
+    onClick={onClick}
+    className={`
+      f6
+      pv3 ph4 ph3-ns
+      ttu
+      no-underline
+      bl bl-0-ns bb-ns bw2 bw1-ns
+      ${isActive
+        ? 'b--orange white'
+        : 'b--transparent white-60'
+      }
```

Comment:
These classes were getting long. I tried breaking each "group" up into its own variable and using `classnames` but ended up using line returns to separate groups and string template expressions for conditionals. Not sure how I feel about this. I like Tachyons but it is definitely different than what I've done in the past (either inline style objects or BEM classes). I like the idea of SRP classes that compose together, but CSS has so many side effects that you end up doing cludgy stuff like this. I think I like it, just need to get used to it :)